docs/create-packages/Creating-a-Package.md

```diff
@@ -180,6 +180,7 @@ The folder conventions are as follows:
 | runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references only for runtime; other files are copied into project folders. There should always be a corresponding (TFM) `AnyCPU` specific assembly under `/ref/{tfm}` folder to provide corresponding compile time assembly. See [Supporting multiple target frameworks](supporting-multiple-target-frameworks.md). |
 | content | Arbitrary files | Contents are copied to the project root. Think of the **content** folder as the root of the target application that ultimately consumes the package. To have the package add an image in the application's */images* folder, place it in the package's *content/images* folder. |
 | build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file or `project.lock.json` (NuGet 3.x+). |
+| buildMultiTargeting | MSBuild `.targets` and `.props` files for cross-framework targeting | Automatically inserted into the project file or `project.lock.json` |
```

Comment:
> project file or `project.lock.json`

We should remove all references to `project.lock.json`. 

It's a leftover from project.json, PackageReference has some differences."

I'd say both build and buildMultiTargeting should say "into the project". 

The file bit is incorrect. We don't really edit the file.

Reply:
We should add an explanation for buildTransitive as well.